pages/index.js

```diff
@@ -1,7 +1,28 @@
+import MovieCard from "@/components/MovieCard";
+import useSWR from "swr";
+
+const fetcher = async (URL) => {
+  const res = await fetch(URL);
```

Comment:
please use full names and no abbreviations for better readability (even though you will see a lot of other code somehwere else)
```suggestion
  const result = await fetch(URL);
```